backend/src/org/akvo/dash/transformation.sql

```diff
@@ -0,0 +1,45 @@
+
+-- :name new-job-execution :! :n
```

Comment:
should this be renamed to something like `new-transformation-job-execution` since we're setting the type to `TRANSFORMATION`? Or, take the type as argument.